src/apify/_actor.py

```diff
@@ -50,6 +51,26 @@
 
 
 MainReturnType = TypeVar('MainReturnType')
+TFun = TypeVar('TFun', bound=Callable[..., Any])
+
+
+def _add_local_storage_error_hint(function: TFun) -> TFun:
+    """This decorator adds a local storage error hint in situation where storage was not found locally."""
+
+    @functools.wraps(function)
+    async def wrapper(
+        self: _ActorType, *, id: str | None = None, name: str | None = None, force_cloud: bool = False
+    ) -> Any:
+        try:
+            return await function(self=self, id=id, name=name, force_cloud=force_cloud)
+        except Exception as e:
+            if not force_cloud:
+                e.args = (
+                    f'{e.args[0]} (If you are trying to retrieve a remote storage, use `force_cloud=True` argument.)',
+                )
```

Comment:
Couldn't `e.args` contain more than one item? If so, am I correct that we are discarding them?